sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/valueentity/ValueEntity.java

```diff
@@ -22,13 +22,34 @@
 import kalix.javasdk.SideEffect;
 import kalix.javasdk.StatusCode;
 import kalix.javasdk.action.Action;
+import kalix.javasdk.eventsourcedentity.EventSourcedEntity;
 import kalix.javasdk.impl.valueentity.ValueEntityEffectImpl;
 import io.grpc.Status;
 
 import java.util.Collection;
 import java.util.Optional;
 
-/** @param <S> The type of the state for this entity. */
+/**
+ * Value Entities persist their state on every change. You can think of them as a Key-Value entity where
```

Comment:
I would love if we call them "KeyValueEntity" instead of "ValueEntity".  I think is more expressive.

Reply:
We have been discussing renaming it. But for the moment, we will start by better documenting it.